lib/dotcom/trip_plan/map.ex

```diff
@@ -37,6 +37,53 @@ defmodule Dotcom.TripPlan.Map do
     |> MapData.add_polylines(paths)
   end
 
+  @doc """
+  Given an `itinerary_map`, convert the polylines to lines (for maplibre-gl-js).
+
+  This involves changing some key names and inverting lat/long to long/lat.
+  """
+  def get_lines(itinerary_map) do
+    itinerary_map
+    |> Map.get(:polylines, [])
+    |> Enum.map(&polyline_to_line/1)
+  end
+
+  @doc """
+  Given an `itinerary_map`, convert the markers to points (for maplibre-gl-js).
+
+  This just gets the longitude and latitude from each marker.
+  """
+  def get_points(itinerary_map) do
+    itinerary_map
+    |> Map.get(:markers, [])
+    |> Enum.map(&marker_to_point/1)
+    |> Enum.reject(&is_nil/1)
+  end
+
+  defp invert_coordinates(coordinates) do
+    coordinates
+    |> Enum.map(&invert_coordinate/1)
+    |> Enum.reject(&is_nil/1)
+  end
+
+  defp invert_coordinate([a, b]), do: [b, a]
+
+  defp invert_coordinate(_), do: nil
```

Comment:
Is this necessary?

Reply:
If we ever get a coordinate that isn't a tuple it will crash the process otherwise.